suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -246,15 +263,47 @@ export const getAccountTypeTech = (path: string) => {
     return 'TR_ACCOUNT_TYPE_BIP44_TECH';
 };
 
-export const getAccountTypeDesc = (path: string) => {
+type getAccountTypeDescProps = {
+    path: string;
+    accountType?: string;
+    symbol?: string;
+    network?: string;
```

Comment:
can you type it properly, not just as strings?

Reply:
Somehow previously I couldn't import the types, but now it's ok. Typed